src/www/ui/admin-dashboard.php

```diff
@@ -207,21 +183,16 @@ function DatabaseQueries()
     $head4 = _("Elapsed");
     $V .= "<tr><th>$head1</th><th>$head2</th><th>$head3</th><th>$head4</th></tr>\n";
 
-    /**** Version ****/
-    $sql = "SELECT * from version();";
-    $result = pg_query($PG_CONN, $sql);
-    DBCheckResult($result, $sql, __FILE__, __LINE__);
-    $row = pg_fetch_assoc($result);
-    pg_free_result($result);
-    $version = explode(' ', $row['version'], 3);
-
     // Get the current query column name in pg_stat_activity
+
+    
+    $sql = "SELECT $1 processid, $2, query_start, now()-query_start AS elapsed FROM pg_stat_activity WHERE $2 != '<IDLE>' AND datname = 'fossology' ORDER BY $1"; 
```

Comment:
* $current_query must not be inserted as $2, otherwise it will be the string instead of column name
* $procpid must not be ...